### PR TITLE
[16634] Folder structure and readme

### PR DESCRIPTION
**Pull request by MRicoIE2C**
Initial confection of the repository's folder structure, library's folder structure, CMakeLists.txt and README files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,21 @@
-# Fast-DDS-QoS-Profiles-Manager
\ No newline at end of file
+# Fast-DDS-QoS-Profiles-Manager
+
+This repository holds a set of tools for the generation of XML files containing Fast DDS profile definitions.
+
+The set includes, under `lib/` folder, the library that generates the XML profile files, exposing APIs to configure the desired profiles.
```
Comment (JesusPoderoso) [inline README.md:5]: I guess there will be only one API with multiple calls (?)
```suggestion
The set includes, under `lib/` folder, the library that generates the XML profile files, exposing API calls to configure the desired profiles.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,21 @@
-# Fast-DDS-QoS-Profiles-Manager
\ No newline at end of file
+# Fast-DDS-QoS-Profiles-Manager
+
+This repository holds a set of tools for the generation of XML files containing Fast DDS profile definitions.
```
Comment (JesusPoderoso) [inline README.md:3]:
```suggestion
eprosima Fast DDS QoS Profiles Manager is a set of tools for the generation of XML files containing Fast DDS profile definitions.
```

File: README.md [hunk 3]
```diff
@@ -1 +1,21 @@
...
+
+The set includes, under `lib/` folder, the library that generates the XML profile files, exposing APIs to configure the desired profiles.
+Under `cli/` folder, a command line interface library provides the possibility of setting simple instructions via command line to configure the desired profiles.
+Under `gui/` folder, a graphical user interface provides that, in order to graphically configure the desired profiles while whatching the changes interactively before saving the XML file.
```
Comment (JesusPoderoso) [inline README.md:7]:
```suggestion
Under `gui/` folder, a graphical user interface allows to graphically configure the desired profiles in a simple form layout. The interface guides the user to complete the profile data, highlighting missing fields and displaying their default values.
```

**Review by JesusPoderoso — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (MRicoIE2C) [reply README.md]: Done

**Pull request merged**